use_cases/Land_Use_Land_Cover/METplus/GridStat_refc_lulc.conf

```diff
@@ -0,0 +1,119 @@
+[config]
+
+# For additional information, please see the METplus Users Guide.
+# https://metplus.readthedocs.io/en/latest/Users_Guide
+
+###
+# Processes to run
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#process-list
+###
+
+#PROCESS_LIST = GridStat, UserScript(metplotpy)
```

Comment:
Should GridStat actually be run in this use case? If so, uncomment this line and remove the next line.